README.md

```diff
@@ -1,77 +1,31 @@
-```
+The recommended way to install and run this repo is using Docker.
 
-To examine MySQL directly you can use docker-compose to exec the mysql client;
+The simplest way to install Docker is to install [the Platform from the official website](https://www.docker.com/products/docker). The official website offers [a good tutorial](https://docs.docker.com/engine/getstarted/).
 
-```bash
-docker-compose exec db mysql -u deltanyc -ppassword deltanyc
-```
+Before launching any of our containers, you'll need to create a .env file. The required fields are listed in .env.example. The defaults are likely good enough for running an instance of the site locally, but you'll need to obtain [an API key for the NYC geoclient](https://developer.cityofnewyork.us/api/geoclient-api) for some of the data cleaning steps to function.
 
-OR you can connect to container over 3306 using the local mysql client if you
-have it installed. See [this article](http://stackoverflow.com/a/32361238/103315)
-on why you need to use `127.0.0.1`.
+`docker-compose up` will install and launch all our containers.  These containers are:
 
-```bash
-mysql -h 127.0.0.1 -P 3306 -u deltanyc -ppassword deltanyc
-```
+* db: The MySQL database storing NYC city data both in semi-raw form and joined into a model for the Rails website. The database can be examined with `docker exec deltanycheatseek_db_1 mysql -u <user from .env> -p <password from .env>`. Local clients can connect at 127.0.0.1:3306 ([localhost will not work](http://stackoverflow.com/a/32361238/103315)).
+* db-udf: Installs [user-defined functions for making regular expression queries](https://github.com/mysqludf/lib_mysqludf_preg) required for data-cleaning into MySQL. Only needs to be run once.
+* nb: Installs Python and starts a Jupyter notebook at [localhost:8888](http://localhost:8888). Can also be used to run the data-import scripts: run `docker exec -it deltanycheatseek_nb_1 /bin/bash`, then execute the scripts
+* web: Installs Ruby on Rails and starts a server at [localhost:3000](http://localhost:3000). Like the data-import scripts, you can conect to a running instance with `docker exec -it deltanycheattseek_web_1 /bin/bash` to run rake tasks. To view a demo of the tool, connect to the instance, then run `rake db:setup`. In the demo, fake data will be provided for "10 West 109th Street".
 
-A note on preg UDFs
-***************************
-The cleanup utils make extensive use of [User Defined preg functions](https://github.com/mysqludf/lib_mysqludf_preg).
-These aren't available in the general MySQL docker container, and the container
-provided by the [uqlibrary](https://github.com/uqlibrary/docker-mysql-udf-preg)
-hasn't been built in a while, so it doesn't have recent fixes to the upstream
-MySQL container.
+Once everything is installed, only the nb or web containers need to be explicitly lanched (eg. `docker-compose up web`) depending on the type of work you plan to do. Launching either will also launch the db container.
```

Comment:
I mean `docker-compose up`: ie the individual containers / images are all installed. The language is a little clunky. Is there better terminology for Docker? Any language suggestions?